content/blog/digidrug_seminar_series.md

```diff
@@ -0,0 +1,19 @@
+---
+title: SAVE THE DATE 09.09.21 - Digital Science for Drug Discovery
+date: 2021-08-10T16:00:00+02:00
+author: corey.taylor
+---
+
+Save the date (Thu. 9. September 2021, 4pm - 5:30pm CET) for an upcoming virtual seminar series, [Digital Science for Drug Discovery](http://www.digidrug.net/). 
+
+Focused on the Berlin region, the series aims to facilitate and enmesh relationships between researchers working within both academia and industry. The over-arching theme? Making efficient and creative use of the wealth of available and growing chemical and biological data combined with powerful computational means at our disposal.
+
+Confirmed speakers include [Dr. Mohammed AlQuraishi](https://systemsbiology.columbia.edu/faculty/mohammed-alquraishi) of [Columbia University](https://systemsbiology.columbia.edu/) and [Dr. Patrick Walters](https://relaytx.com/our-team/pat-walters-ph-d/) of [Relay Therapeutics](https://relaytx.com/our-team/pat-walters-ph-d/). 
```

Comment:
Just realized that the last two links are the same.

Reply:
Yup, fixed (lazy editing on my part)